Anchor.toml

```diff
@@ -5,7 +5,7 @@ resolution = true
 skip-lint = false
 
 [programs.localnet]
-degen_pools = "2JWqYTXG5yHSU78hjKb39YFx82whbK74v6sMqMG3TVBQ"
+degen_pools = "FkLPinEx3vxjjLv2PxNTb7BC4wU2ohi2yd28SwfhqNXh"
```

Comment:
We should not push the change of the program id this will replace the one in production